src/app/protocol.ts

```diff
@@ -223,14 +176,14 @@ class XpraProtocol {
     while (this.websocket && this.do_process_receive_queue());
   }
 
-  error() {
+  error(...args) {
```

Comment:
Is this a good practice? ....args is defined but not used.

Reply:
Well, we should use `...args` instead of `arguments`. using `...args` tells TypeScript the signature of the method should take any number of arbitrary inputs